source/frontend/src/models/generate/acquisition/GenerateAcquisitionFile.ts

```diff
@@ -32,7 +33,7 @@ export class Api_GenerateAcquisitionFile {
   project_number: string;
   project_name: string;
   prov_solicitor?: Api_GeneratePerson;
-  owner_solicitor?: Api_GeneratePerson;
+  owner_solicitor?: Api_GenerateInterestHolder;
```

Comment:
Owner solicitor is an interest holder - it can be a person or an organization (with optional primary contact)